_posts/2023-10-17-cmor3.md

```diff
@@ -0,0 +1,22 @@
+---
+title:  "CMOR 3.7.3 released!!!"
+published: true
+permalink: /news/:year/:month/:day/:title/
+tags: [news, getting_started]
+---
+
+Changes
+* CMOR now supports Python 3.12.
+
+Bugfixes
+* Fixed error caused when `cmor_write` writes arrays with more than 2<sup>31</sup>-1 elements.
+* Fixed segmentation fault casued when the value for `further_info_url` in the user input is an empty string or only whitespace.
+* Fixed implicit declaration compilation error in cmor.c.
```

Comment:
`cmor.c`?

Reply:
It's a fix of a compile error in a C source file.